src/focus.js

```diff
@@ -48,8 +49,7 @@ export default {
             focusChain = []
           }
         },
-        // todo: make the hold timeout configurable?
-        this.hold ? 50 : 0
+        this.hold ? Settings.get('focusHoldTimeout') : 0
```

Comment:
Here we need to keep the default of 50 when the setting is not provided by the user. We can do this by passing a second argument to `Settings.get()`

Reply:
Done